pyproject.toml

```diff
@@ -73,17 +73,11 @@ notebooks = [
   "ipysheet"
 ]
 
-[tools.setuptools]
```

Comment:
My understanding is that this is required to include files that have extensions other than `.py` in the package when it is built. Its then configured in the other section below.